### PR TITLE
Implement support for Portworx's per-usage licensing

### PR DESCRIPTION
**Pull request by logdnalf**
This PR implements support for enabling Portworx's per-usage licensing, which is enabled by specifying an account key (a string literal) that is stored inside a Kubernetes Secret.

### CODE REVIEW

File: stable/index.yaml [hunk 1]
```diff
@@ -11,12 +23,12 @@ entries:
     sources:
     - https://github.com/portworx/helm
     urls:
-    - https://raw.githubusercontent.com/portworx/helm/master/stable/portworx-1.0.0.tgz
+    - https://github.com/logdnalf/helm/blob/per_usage_licensing/stable/portworx-1.0.0.tgz
```
Comment (disrani-px) [inline stable/index.yaml:26]: These need to point to the portworx repo and not the fork

Comment (logdnalf) [reply stable/index.yaml]: Fixed.

**Comment by logdnalf** [comment]
@disrani-px please take another look

**Review by disrani-px — APPROVED** [review]

**Comment by disrani-px** [comment]
@logdnalf Can you squash the commits into one?

**Comment by logdnalf** [comment]
@disrani-px done: all commits squashed into one. Feel free to merge at your leisure.

**Pull request merged**